### PR TITLE
Update selenium version

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Part of https://github.com/dotnet/aspnetcore-internal/issues/3777

I'm hoping we can move to using Playwright asap so we don't have to make these types of changes.

### CODE REVIEW

**Review by dougbu — CHANGES_REQUESTED** [review]
@javiercn I don't see anything using the `$(Selenium*)` properties. Did you leave them behind for a reason❔ If not, can anything else from our Selenium history that could be deleted❔

@JunTaoLuo I suggest repurposing this PR to _remove_ the `$(Selenium*)` properties and whatever else @javiercn says can go.

Please also update the ops rotation docs and issue template. @javiercn should something be done with the `$(Playwright*)` properties on a regular basis❔

**Comment by JunTaoLuo** [comment]
I was wondering about that since I still see Selenium used in https://github.com/dotnet/aspnetcore/blob/main/src/Servers/Kestrel/test/Interop.FunctionalTests/Interop.FunctionalTests.csproj#L27-L28. I think only the blazor template tests and some other tests have been converted to using Playwright, not the entire repo.

**Comment by javiercn** [comment]
@dougbu Components still uses Selenium.

**Comment by dougbu** [comment]
More generally, any changes expecting to grab new versions from NuGet must be coordinated with @mmitche, @wtgodbe, or DncEng. Must get the new packages into the `dotnet-public` feed prior to submitting PRs like this one.

**Comment by JunTaoLuo** [comment]
Yup, I just remembered we don't get packages from NuGet directly.

**Comment by dougbu** [comment]
Ah, my bad on thinking the properties should be removed. Do we _also_ need to get new Playwrite versions❔

**Comment by javiercn** [comment]
@dougbu to a Degree, but we are still figuring out how often we want to update. 

Playwright releases at the same cadence as Chrome, but a given version targets a fixed version of Chrom(ium) so as long as we don't stay behind for a year we are likely ok. We can always target a new browser version with an older driver, but compatibility is not guaranteed in that case.

**Comment by JunTaoLuo** [comment]
I think that was the rationale for us to upgrade Selenium versions as part of Ops rotation. It's a preventative measure so that we are not blindsided by any chrome/driver incompatibilities. Is there harm in updating too frequently @javiercn ?

File: eng/Versions.props [hunk 1]
```diff
@@ -247,10 +247,10 @@
     <PlaywrightSharpVersion>0.180.0</PlaywrightSharpVersion>
     <PollyExtensionsHttpVersion>3.0.0</PollyExtensionsHttpVersion>
     <PollyVersion>7.1.0</PollyVersion>
-    <SeleniumSupportVersion>4.0.0-alpha07</SeleniumSupportVersion>
+    <SeleniumSupportVersion>4.0.0-beta1</SeleniumSupportVersion>
     <SeleniumWebDriverMicrosoftDriverVersion>17.17134.0</SeleniumWebDriverMicrosoftDriverVersion>
```
Comment (dougbu) [inline eng/Versions.props:251]: This property is no longer used. Can also remove Selenium.WebDriver.MicrosoftDriver from Dependencies.props.

File: eng/Versions.props [hunk 2]
```diff
@@ -247,10 +247,10 @@
...
     <SeleniumWebDriverChromeDriverVersion>87.0.4280.2000-beta</SeleniumWebDriverChromeDriverVersion>
```
Comment (dougbu) [inline eng/Versions.props:252]: Current Chrome driver version is `89.0.4389.2300-beta`. Are we unable to update to match what we're likely downloading on the CI agents❔

**Comment by dougbu** [comment]
> It's a preventative measure so that we are not blindsided by any chrome/driver incompatibilities.

Agreed. We don't have control over the Chrome version eng\scripts\InstallGoogleChrome.ps1 installs because the download URI is not version-specific.

File: eng/Versions.props (hunk 1 above)
Comment (JunTaoLuo) [reply eng/Versions.props:251]: Good catch.

File: eng/Versions.props (hunk 2 above)
Comment (JunTaoLuo) [reply eng/Versions.props:252]: I was trying to remember how this version relates to https://github.com/dotnet/aspnetcore/blob/main/src/Shared/E2ETesting/selenium-config.json#L4. After looking through commits, I think it's safe to update this and then change the selenium-config.json to match it as needed.

**Comment by javiercn** [comment]
> Agreed. We don't have control over the Chrome version eng\scripts\InstallGoogleChrome.ps1 installs because the download URI is not version-specific.

Chrome doesn't offer version specific stuff for these things, once a new version is out is "impossible" to download the older version bits (I tried at some point).

**Comment by JunTaoLuo** [comment]
FYI, I'll edit the doc to include updating Playwright when this PR passes.

**Review by dougbu — APPROVED** [review]

**Comment by JunTaoLuo** [comment]
Hmmm there was a complication with the playwright version update complaining that the driver executable can’t be found. I’ve reverted that to unblock the selenium update.

**Pull request merged**